modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -37,6 +38,56 @@
+
+Server 10.11 introduces the ability to share with multiple users at once by entering their identifiers (e.g., display name, user ID, email address, federated cloud ID) as a comma-separated list in the sharing dialog (e.g., `alice,bob@example.org,carol`. Based on the inputs, ownCloud will decide how to create the shares (e.g., share with an internal user, invite a new guest, create a federated share). This way, users can easily invite many recipients at once to their files and folders. With Custom Groups version 0.7.0 onwards the same behavior also applies to adding users to Custom Groups. https://github.com/owncloud/core/pull/40155[#40155] https://github.com/owncloud/customgroups/pull/522[customgroups#522] https://github.com/owncloud/guests/pull/506[guests#506]
+
+=== Inviting New Guests to Custom Groups
+
+Server 10.11 in combination with Custom Groups 0.7.0 and Guests 0.11.0 allows inviting new guest users directly into a Custom Group. Previously this was only possible by sharing files with new guests first and creating their user account thereby. New guests will not have access to any files nor will they be able to create/upload their own files unless there is a share with the respective Custom Group or with the guest user. https://github.com/owncloud/customgroups/pull/521[#521] https://github.com/owncloud/guests/pull/506[guests#506]
+
+=== New skeleton files for users
+
+For new user accounts, the default skeleton files have been changed a bit in version 10.11. The ownCloud user manual PDF has been removed as it becomes outdated quickly and will not be updated for existing users when a server upgrade is done. It is recommended to provide the official user documentation to users instead. New users will further find a folder called "Learn more about ownCloud" that contains information about the product.
+
+=== Notable Changes
+
+* The default app on login can now be set per individual user. ownCloud Web will use this to remember which UI the user has used during their last session. This way, users are always redirected to their preferred UI after logging in. https://github.com/owncloud/core/pull/39600[#39600]
+* ownCloud Web will now respect the user's language setting. https://github.com/owncloud/core/pull/40087[#40087]
+* The command `occ background:queue:status` will now output more information about background jobs. https://github.com/owncloud/core/pull/40113[#40113]
+* Quota handling has been improved (e.g., when copying or uploading files into shared folders). https://github.com/owncloud/core/pull/39895[#39895]
+* The context menu for multiple file actions in public links (e.g., "Open with PDF Viewer") works again. https://github.com/owncloud/core/pull/40143[#40143]
+* Storage encryption has received stability improvements. https://github.com/owncloud/core/pull/40240[#40240] 
+* Guest users will no longer see settings sections which they are not allowed to use. https://github.com/owncloud/core/pull/40257[#40257]
```

Comment:
Nope. My guest user always sees customsgroups. whitelisted or not.
https://github.com/owncloud/guests/issues/516#issuecomment-1241300003

Reply:
See guests PR https://github.com/owncloud/guests/pull/518 - that is the missing piece for this guest whitelist fix. We need a guests app release to go with core 10.11

Reply:
Much better! thanks!